src/components/tables/tests/TableConnected.spec.tsx

```diff
@@ -53,6 +76,24 @@ describe('<TableConnected />', () => {
             tableConnected.unmount();
             expect(store.getState().tables[tablePropsMock.id]).toBeUndefined();
         });
+
+        it('should update action in props with the selectedIds array and the rowsData from the state', () => {
+            const actions = [{id: 'action1', enabled: true}];
+            const wrapper = mountComponentWithProps({...tablePropsMock, actionBar: true, getActions: () => actions, initialTableData: getInitialTableDataMock()});
+
+            const tableConnected = wrapper.find(Table);
+
+            expect(tableConnected.props().actions[0]).toBe(actions[0]);

```

Comment:
could you do `expect(tableConnected.props().actions).toEqual(actions);` ? and since this is an object equality i think you should use `toEqual` IIRC